tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md

```diff
@@ -0,0 +1,131 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product-function>ui-sap-business-client--nwbc-]
+primary_tag: software-product-function>ui-sap-business-client--nwbc-
+author_name: Sandra Thimme
+author_profile: https://github.com/sandra36
+---
+
+# Launch native SAP GUI for Windows Applications in the Cloud with SAP Business Client
+<!-- description --> Running SAP Build Work Zone, standard edition inside SAP Business Client you can mix native SAP GUI transactions with state-of-the-art web applications. Therefore, a TCP ping to the remote system has to be performed at the time the application shall be started.
+
+## You will learn
+- What is SAP Build Work Zone, standard edition?
+- What are the Integration Prerequisites?
+- What is the difference between direct and tunneled Access?
+- How to run a cloud launchpad within SAP Business Client in 3 steps.
+
+## Prerequisites
+- You have installed SAP Business Client 7.70.
+SAP ONE Support Launchpad ([SAP ONE Support Launchpad](https://launchpad.support.sap.com/#/softwarecenter/template/products/%20_APP=00200682500000001943&_EVENT=DISPHIER&HEADER=Y&FUNCTIONBAR=N&EVENT=TREE&NE=NAVIGATE&ENR=73554900100200012051&V=MAINT&TA=ACTUAL&PAGE=SEARCH/SAP%20BUSINESS%20CLIENT%207.70))
+- You have created a **trial account on SAP BTP**:  [Get a Free Account on SAP BTP Trial](hcp-create-trial-account)
+- You are subscribed to the SAP Build Work Zone, standard edition including the Launchpad Admin role: see the following tutorial [Deliver Your First SAP Fiori Launchpad Site] (mission.cp-starter-digitalexp-portal)
+
+See also: [SAP Build Work Zone, standard edition (formerly known as SAP Launchpad Service) on BTP](https://help.sap.com/docs/Launchpad_Service/8c8e1958338140699bd4811b37b82ece/fd79b232967545569d1ae4d8f691016b.html?locale=en-US)
+
+## Intro
```

Comment:
This should be a more detailed version of what this tutorial is about. The first paragraph is listing more prerequisites, which should be added to the prerequisites section.

Also, do not use the "royal we" unless you are referring to SAP.